ext/pdo_pgsql/pdo_pgsql.c

```diff
@@ -156,6 +156,9 @@ PHP_MINIT_FUNCTION(pdo_pgsql)
 	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_TRANSACTION_INTRANS", (zend_long)PGSQL_TRANSACTION_INTRANS);
 	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_TRANSACTION_INERROR", (zend_long)PGSQL_TRANSACTION_INERROR);
 	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_TRANSACTION_UNKNOWN", (zend_long)PGSQL_TRANSACTION_UNKNOWN);
+#ifdef HAVE_PG_RESULT_MEMORY_SIZE
+	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_ATTR_RESULT_MEMORY_SIZE", (zend_long)PRO_PGSQL_ATTR_RESULT_MEMORY_SIZE);
```

Comment:
typo?

```suggestion
	REGISTER_PDO_CLASS_CONST_LONG("PGSQL_ATTR_RESULT_MEMORY_SIZE", (zend_long)PDO_PGSQL_ATTR_RESULT_MEMORY_SIZE);
```

Reply:
Thsnks!! 🫠🫠🫠
Fixed in https://github.com/php/php-src/pull/14260/commits/a420d3afda36fc0789dfd7e56deb72d37ae3854e